tests/test_integration.py

```diff
@@ -1640,11 +1668,11 @@ def create_pex(path, seed):
             zf1.extractall(path=unzipped1)
             zf2.extractall(path=unzipped2)
             for member1, member2 in zip(sorted(zf1.namelist()), sorted(zf2.namelist())):
-                assert filecmp.cmp(
-                    os.path.join(unzipped1, member1),
-                    os.path.join(unzipped2, member2),
-                    shallow=False,
-                )
+                member1_path = os.path.join(unzipped1, member1)
+                if os.path.isdir(member1_path):
+                    continue
```

Comment:
What's the reason for skipping? Are the directory entries not byte-for-byte identical?

Reply:
It's because `filecmp.cmp` considers anything not a regular file (symlink, directory, etc.) unequal.